src/vs/base/common/color.ts

```diff
@@ -399,6 +399,22 @@ export class Color {
 		return new Color(new RGBA(r, g, b, a));
 	}
 
+	toRGB(...backgrounds: Color[]): Color {
```

Comment:
@Dari-K should this have a better name to reflect what it does? It seems to blend multiple colors into one?

Reply:
Sure, can change the name, not entirely sure what would fit though, because blend would imply to me that it's a mix of the colours (if I did blend on red and blue I would expect to get purple), but if the foreground colour is opaque, it will be returned unchanged. It's more like a flatten operation. What do you think?

Reply:
@Dari-K yeah better 👍